.mergify.yml

```diff
@@ -0,0 +1,26 @@
+queue_rules:
+  - name: default
+    conditions: []
+
+pull_request_rules:
+  - name: Automatic merge on approval
+    conditions:
+      - "base=main"
+      - "#approved-reviews-by>=1"
```

Comment:
The condition is relaxed, but I can't find a better condition. IMO, there should a label "status/can-merge" and labeled by reviewer or author manually.

Reply:
Doesn't mergify support label?

Reply:
It supported, but should we add label manually?

Reply:
Or introduce another label management action?

Reply:
I think we can have a bot or GitHub Action that:

* When we type `/merge` in comment, a `can-merge` label will be added.
* Then Mergify will attempt to get the PR merged.

Reply:
I think /merge is not simpler than adding the label manually? (Btw, would you want /lgtm? 🥸

Reply:
LGTM is same as “approve”, I think it’s different from merge.

Reply:
I mean the automation seems unnecessary. e.g., using approve button is enough, no need to introduce /lgtm automation. Same for /merge